src/features/settings/components/ExportSelectionModal/ExportSelectionModal.module.css

```
@@ -43,12 +43,38 @@
 .sectionList {
   display: flex;
   flex-direction: column;
-  gap: var(--spacing-sm);
-  max-height: 300px;
+  gap: var(--spacing-md);
+  max-height: 400px;
   overflow-y: auto;
   padding: var(--spacing-sm) 0;
 }
 
+.settingsItem {
```

Stylelint (17.2.0)

[error] Expected class selector ".settingsItem" to be kebab-case (selector-class-pattern)

(selector-class-pattern)